rpc/handler.go

```diff
@@ -40,6 +41,8 @@ import (
 	"golang.org/x/time/rate"
 )
 
+const truncateLimit = 1024
```

Comment:
this should be around 4 kb, seemed a reasonable limit to me. open for negotiations :)

Reply:
(style) I think that given the distance from where it's used, the name should be more informative; perhaps `callErrorLogTruncateLimit`. Alternatively, just declare it closer to where it's used.

[See style guide](https://google.github.io/styleguide/go/decisions#:~:text=The%20general%20rule%20of%20thumb%20is%20that%20the%20length%20of%20a%20name%20should%20be%20proportional%20to%20the%20size%20of%20its%20scope)

Reply:
renamed 👍